README.md

```diff
@@ -408,6 +408,10 @@ Function which takes an object with the current route and returns a test id for
 />
 ```
 
+##### `tabBarTestID`
```

Comment:
Let's rename the prop to `testID`

Also, move this section towards the end. The documentation is organized to have functions together

Reply:
Thanks for pointing out, I've made the suggested changes.